contracts/IAssetsExchange.sol

```diff
@@ -48,5 +48,28 @@ interface IAssetsExchange {
   function getMinimumERC20TokenAmountForExactAVAX(uint256 _exactAmountOut, address _token) external returns(uint256);
 
 
+  /**
+     * Adds or updates a supported asset
+  **/
+  function setAsset(bytes32 _asset, address _address) external;
```

Comment:
`updateAsset` method is not included, since `setAsset` offers this functionality already